var/spack/repos/builtin/packages/samrai/package.py

```diff
@@ -89,6 +96,10 @@ def configure_args(self):
             '--with-hypre=no',
             '--with-petsc=no'])
 
+        # SAMRAI 2 used templates; enable implicit instantiation
+        if self.version == Version('2.4.4'):
```

Comment:
I know 2.4.4 is the only 2 currently in Spack, but just in case someone adds another, can you change this to:
```python
if self.spec.satisfies('@:2'):
```

Reply:
You meant `'@:3'`, right?